### ISSUE TITLE
 Could not parse image id

### ISSUE THREAD

**Issue by EASTCATV** [marcuslonnberg/sbt-docker#125]
java.lang.RuntimeException: Could not parse image id
        at scala.sys.package$.error(package.scala:27)
        at sbtdocker.DockerBuild$.build(DockerBuild.scala:99)
        at sbtdocker.DockerBuild$.buildAndTag(DockerBuild.scala:72)
        at sbtdocker.DockerBuild$.apply(DockerBuild.scala:45)
        at sbtdocker.DockerBuild$.apply(DockerBuild.scala:24)
        at sbtdocker.DockerSettings$$anonfun$baseDockerSettings$1.apply(DockerSettings.scala:17)
        at sbtdocker.DockerSettings$$anonfun$baseDockerSettings$1.apply(DockerSettings.scala:10)
        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
        at sbt.std.Transform$$anon$4.work(System.scala:63)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
        at sbt.Execute.work(Execute.scala:235)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
[error] (*:docker) Could not parse image id

**Comment by marcuslonnberg** [comment]
@EASTCATV would you mind providing context to this issue? Such as which version of sbt-docker and docker is used, OS, Dockerfile, etc.

**Comment by oms** [comment]
I can't speak for @EASTCATV, but I do notice that you run into the same problem, if you for example were to point to `podman` instead of the `docker` binary.

The output from a succesful build command contains an extra line at the end with the full image id, like so:

```
$ podman build -t testbuild .
[…snip…]
--> dada5485d85
Successfully tagged localhost/testbuild:latest
dada5485d85618e75ad1a9772c6c00523f442c8d30487fb7c9f9f9ea544db1db
```

This won't yield a match  with the `lastOption` on `processOutput.collect`. Would be nice to get the podman variant supported, because I suspect compatibility otherwise is pretty on par.

**Comment by marcuslonnberg** [comment]
Thanks for your comment @oms! I have now released version 1.8.3 which I hope solves the problem.
